plugins/de.ovgu.featureide.fm.core/src/de/ovgu/featureide/fm/core/base/IFeatureStructure.java

```diff
@@ -119,7 +127,13 @@
 
 	void setParent(IFeatureStructure newParent);
 
+	void setAttributeListInherited(LinkedList<FeatureAttributeInherited> attListRecursive);
+
+	void setAttributeList(LinkedList<FeatureAttribute> attList);
+
 	void setRelevantConstraints();
 
 	void setRelevantConstraints(List<IConstraint> constraints); // Marcus, if calculated outside the class, see FeatureUtils.setRelevantConstraints(...)
+
+	LinkedList<FeatureAttribute> getRecursiveList();
```

Comment:
Please provide a better name e.g. getAttributeListRecursive or something like this. This minimizes possible confusions.